charts/rancher-istio/v1.1.1/charts/prometheus/templates/service.yaml

```diff
@@ -14,15 +14,31 @@ metadata:
     heritage: {{ .Release.Service }}
     release: {{ .Release.Name }}
 spec:
-  type: {{ .Values.service.type }}
+  type: ClusterIP
+  selector:
+    app: prometheus
   ports:
-  - name: http-prometheus
-    targetPort: 9090
```

Comment:
Use this name because istio encourages us set service ports name start with the protocol, I fellow this to makes it easy if we want istio to manage this service later. But we haven't use it yet.